core/common/src/main/java/alluxio/wire/InconsistentProperty.java

```diff
@@ -34,6 +34,16 @@
    */
   public InconsistentProperty() {}
 
+  /**
+   * Creates a new instance of {@link InconsistentProperty} from thrift representation.
+   *
+   * @param inconsistentProperty the thrift inconsistent property
+   */
+  protected InconsistentProperty(alluxio.thrift.InconsistentProperty inconsistentProperty) {
```

Comment:
protected -> public?

Reply:
Our conventions seem to make thrift constructor and toThrift function protected in wire type and all the other places need to use ThriftUtils.fromThrift and ThriftUtils.toThrift.

Reply:
got it, lets stick with the convention then